chapters/10-community-involvement.Rmd

```diff
@@ -9,47 +9,101 @@
+        - [Tidyverse](https://www.tidyverse.org/contribute/)
+    - The default `.github/CONTRIBUTING.md` file you created has a template for
+    instructions for submitting changes or reporting a problem. Modify
+    the `CONTRIBUTING.md` file to include the following:
+        - A welcome statement and a brief description of your project.
+        - A list of project-related resources, like a link to your README.
+        - A list of communication channels your project uses, if applicable
+        (mailing lists, forums, chat clients, GitHub Issues, etc.)
+        TODO: The above might be more appropriate for the project management
+        section.
+        - Instructions for how to submit changes or report a problem, specific
+        to your project.
+        - A description of how you recognize contributors.
+        - Instructions on where to go for help: who to contact and how.
+        - A link to your Code of Conduct.
  
-4. Add a license 
-- Choose a license for your package. Visit [choosealicense.com](https://choosealicense.com/) and choose a license that makes sense for your project. This license will live in a file called LICENSE.md in the top level of your project. If you choose a common license, it's likely that you can create LICENSE.md by running `usethis::use_mit_license()` (for the MIT license) or another related `use_NAME_license` function.
+4. Add a license to your repository for terms of re-use of your package:
+    - Choose a license for your package. Visit
+    [choosealicense.com](https://choosealicense.com/) and choose a license that
+    makes sense for your project. This license will live in a file called
+    `LICENSE.md` in the top level folder of your project. If you choose a common
+    license, it's likely that you can create `LICENSE.md` by running
+    `usethis::use_mit_license()` (for the MIT License) or another related
+    `usethis::use_NAME_license()` function. Licensing is a complex thing
+    so for now, choose the MIT License.
+
+5. Project Governance:
+    - FIXME: this might be a good place to go through the 
+    [Personas and Pathways](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/)
+    exercise, then add something to CONTRIBUTING.md about what the leadership
+    structure is, if any, and how to get there.
 
-5. Project Governance
-- FIXME: this might be a good place to go through the [Personas and Pathways](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/) exercise, then add something to CONTRIBUTING.md about what the leadership structure is, if any, and how to get there. 
+TODO: Should this be so late? I know it feels like it fits well here, just putting this out there.
```

Comment:
I don't think it's late. We're assuming the project is developed by a single person since the start of the book, and this chapter is where we start to think about other contributors. I think it's useful to think about personas in other capacities (e.g., who's going to *use* this package), but we're focusing on contributors here.

Reply:
The idea of a "user persona" is a good one, and relates to #75